.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,34 @@
+name: Backend Pull Request
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
```

Comment:
Consider adding `permissions` section to the workflow, in order to limit `GITHUB_TOKEN` access.